IBM Process Mining Trimming/README.md

```diff
@@ -0,0 +1,10 @@
+# IBM Process Mining
+API client samples
+
```

Comment:
Can you please explain that this is valid for Process Mining 1.12.0. Also I would add that trimming means removing from the specified process id all the data that is expressed in the filter.

Reply:
It is not so obvious that the filter expressed the data removed or the data that is kept.